bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/references/ModernizrJavaScriptReference.java

```diff
@@ -31,7 +31,7 @@ public static ModernizrJavaScriptReference instance() {
      * Private constructor.
      */
     private ModernizrJavaScriptReference() {
-        super("/modernizr/current/modernizr.js");
+        super("/modernizr/current/modernizr.min.js");
```

Comment:
Is this really needed ? Wicket should use the min version in PROD mode

Reply:
I have not tested in PROD mode yet, but in DEV mode I get the mentioned exception ...

Reply:
I'd say the real question is: why does the webjars artifact of modernizr-2.8.3-1 not contain modernizr.js? modernizr-2.8.3 contains both (modernizr.js and modernizr.min.js) ...

Reply:
@martin-g what do you think? I currently created a workaround for this in my current project (created my own ModernizrJavaScriptReference with the currected path and configured a global resource replacement in Wicket).